starfish/util/synthesize.py

```diff
@@ -1,203 +1,61 @@
-import random
```

Comment:
This was split up and moved into other modules (Codebook, Stack, IntensityTable). A wrapper around synthetic generator methods in each module replaces the previous code (at the bottom of the diff).